### ISSUE TITLE
Prevent interaction with disabled cells to match Excel behavior

### ISSUE THREAD

**Issue by underlog** [telerik/kendo-ui-core#1750]
Should check and validate this - perhaps it makes sense for us, too.

Reported in Ticket No 1022755.

**Comment by tsvetomir** [comment]
To quote the customer:
> In Excel, user can't select or drag disabled cells.

Excel gives you [fine-grained control](https://support.office.com/en-us/article/Lock-cells-in-a-worksheet-B2F833BD-9DB5-48A4-9EC0-8E451AC27C3A) over what you can do with locked cells. By default, users are allowed to select locked cells.
Unless we implement such fine-grained controls I suggest we stick with this behavior.

I don't see a way to enable Auto-Fill for them and this is probably an odd feature for someone coming from Excel.

@yordanov, @mishoo, @gyoshev, what do you think?

**Comment by mishoo** [comment]
Auto-fill was blocked on disabled cells for https://github.com/telerik/kendo/issues/5589.  More precisely, one can start auto-filling by dragging a disabled cell, but the destination range cannot contain disabled cells.

I'm not sure how easy it would be to implement finer grained control.  The navigation code is already quite tricky…

**Comment by tsvetomir** [comment]
Thanks! I think it's best to leave such fine-grained controls over disabled cells for a separate story.

Auto-filling is disabled and the selection remains enabled.
